CovertInfiltration/Src/CovertInfiltration/Classes/UISSManager_CovertAction.uc

```diff
@@ -80,6 +81,8 @@ simulated protected function BuildConfiguration()
 		// have explicit rank names set up, it'll use the standard code path of falling back to the default ranks.
 		if (StaffSlotState.RequiredMinRank > 0) Slots[i].Notes.AddItem(CreateRankNote(StaffSlotState.RequiredMinRank, StaffSlotState.RequiredClass));
 		
+		if (i == 4 && !`XCOMHQ.HasSoldierUnlockTemplate('InfiltrationSize1')) Slots[i].Notes.AddItem(CreatePenaltyNote());
+		if (i == 5 && !`XCOMHQ.HasSoldierUnlockTemplate('InfiltrationSize2')) Slots[i].Notes.AddItem(CreatePenaltyNote());
```

Comment:
I don't think showing it on concrete slots is a good idea, since it calculates based on total filled slots, remove it. Probably a better idea to show something in the top-left readout when over the current count